pkgs/top-level/beam-packages.nix

```diff
@@ -49,6 +49,10 @@ rec {
     # access for example elixir built with different version of Erlang, use
     # `beam.packages.erlangR19.elixir`.
     elixir = packages.erlang.elixir;
```

Comment:
You can replace these 4 lines with

    inherit (packages.erlang) elixir elixir-1_5 elixir-1_4 elixir-1_3;